api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -356,13 +356,15 @@ public static class BucketAuditConfig {
   }
 
   public static class RdrExportConfig {
-    // RDR Host to connect to to export data
+    // RDR Host to connect to for exporting data
     public String host;
-    // Google cloud Queue name to which the task will be pushed to
+    // Google cloud Queue name where tasks are pushed
+    @Deprecated(forRemoval = true) // hardcoded to RDR_EXPORT_QUEUE_NAME
```

Comment:
we already need to hardcode this queue name (and can't vary it by env) because we define it in queue.yaml